src/ansys/fluent/core/session.py

```diff
@@ -326,11 +326,19 @@ def exit(self, **kwargs) -> None:
         """Exit session."""
         logger.debug("session.exit() called")
         self._fluent_connection.exit(**kwargs)
+        try:
+            self._file_transfer_service.exit()
+        except AttributeError:
+            pass
 
     def force_exit(self) -> None:
         """Immediately terminates the Fluent session, losing unsaved progress and
         data."""
         self._fluent_connection.force_exit()
+        try:
+            self._file_transfer_service.exit()
+        except AttributeError:
+            pass
```

Comment:
Use is allowed to plug in own FTS. If user has not implemented method to exit respected FTS then we will get `Attribute` error. To handle this scenario we have done this.

Reply:
Presumably if users set up their own FTS, then they would have to provide an `exit` method as well, right? Otherwise they will constantly have zombie services running in their machine. 

Doesn't that mean that we don't want to hide an `AttributeError` here, so that the user is aware of this issue?

Reply:
Earlier I had kept it as `self._file_transfer_service.exit()` only without try block. We can raise an exception with more information in `except` block instead of `pass` only.

Reply:
they would have to provide an exit method as well, right? - yes.

Reply:
We have removed try block and user will get `AttributeError` now.

Reply:
We will update `exit` method. Thank you.